src/main/resources/assets/iris/lang/de_de.json

```diff
@@ -15,6 +15,8 @@
   "options.iris.apply": "Anwenden",
   "options.iris.refresh": "Aktualisieren",
   "options.iris.openShaderPackFolder": "Shaderpaket-Ordner öffnen ...",
+  "options.iris.shaderPackSettings": "Shaderpaket-Einstellungen...",
```

Comment:
Please include a space before the `...` at the end. I don't know why we did it that way but Optifine does the same as the previous translation so we should stay consistent